zilliqa/src/api/types/eth.rs

```diff
@@ -24,6 +24,63 @@ pub enum HashOrTransaction {
     Transaction(Transaction),
 }
 
+#[derive(Clone, Serialize)]
+pub struct QuorumCertificate {
+    #[serde(serialize_with = "hex")]
+    pub signature: Vec<u8>,
+    pub cosigned: String,
+    #[serde(serialize_with = "hex")]
+    pub view: u64,
+    #[serde(serialize_with = "hex")]
+    pub block_hash: B256,
+}
+
+impl QuorumCertificate {
+    pub fn from_qc(qc: &message::QuorumCertificate) -> Self {
+        Self {
+            signature: qc.signature.to_bytes(),
+            cosigned: qc
+                .cosigned
+                .clone()
+                .iter()
+                .fold(String::new(), |mut acc, bit| {
+                    acc.push(if *bit { '1' } else { '0' });
+                    acc
+                }),
+            view: qc.view,
+            block_hash: qc.block_hash.into(),
+        }
+    }
+}
+
+#[derive(Clone, Serialize)]
+pub struct AggregateQc {
+    #[serde(serialize_with = "hex")]
+    pub signature: Vec<u8>,
+    pub cosigned: String,
+    #[serde(serialize_with = "hex")]
+    pub view: u64,
+    pub qcs: Vec<QuorumCertificate>,
```

Comment:
Just checking here that the AggregateQc is based on the block hash in the Header it's included in? (otherwise you don't know at which block to `eth_call()` to get your committee) - mention it just because you included the block_hash in the Qc in a way that suggested that the Qc wasn't necessarily ..